### ISSUE TITLE
MFR-005 Print Full Spell Descriptions (possibly with spell cards)

### ISSUE THREAD

**Issue by crawltracker** [flapkan/mpmb-tracker#5]
**Feature Request:** Full Length Spell Descriptions

**Extra Information**
**Who would use it?** People that want to generate spellsheets with complete descriptions to avoid the need to source the rules from other sources, look them up in source books or google them.
**How would it work?** When generating a spellsheet, some sort of "full length descriptions". Instead of an abbreviated description squeezed on to one line, a full spell description spanning multiple lines would be generated. Spellsheets would be longer in general, but give much more information.
**Why should this be added?** It gives users the option to have a more complete spellsheet so that they don't need to crossreference or look up spells anymore. I'd be more convenient for people that want the option.

- smitherin0#0

> Upvote - lorddusk#0

> Upvote - satkaz#0

> Upvote - 189723300522491904

> Upvote - thalimon#0

> Upvote - quagscoped#0

> Upvote - cw830709#0

> Upvote - iamneverdead#0

> Upvote - 149314483737591808

> Upvote - kreekgod#0

> Upvote - darkkijin#0

> Downvote - dorgrin#0

> Upvote - smitherin0#0

> Upvote - 159003537408262144

> Upvote - 309690543380627456

> Upvote - sudsnbubbles#0

> Upvote - aykroyd007#0

> Upvote - baextower#0

> Upvote - 385506396063399947

> Upvote - mick.orc#0

> Upvote - 246707279745908736

> Upvote - polyjohn#0

> Upvote - 119814535614955520

> Upvote - myoukochou#0

> Upvote - ganglari_eeh#0

> Upvote - flamingkilla#0

> Upvote - bartbaardman#0

> Downvote - tikipower#0

> Upvote - insideabasketcase#0

> Upvote - boylane#0

> Upvote - gaminggirl#0

> Upvote - 366978991502000140

> Upvote - crazyfruit13#0

> Upvote - dwinther#0

> Upvote - the-pieman#5849

> Upvote - 140924062959534081

> Downvote - bevan7#0

> Upvote - seasandz#0

> Downvote - john_hos#0

> Upvote - joshy_13#0

> Downvote - 344548722442633226

> Upvote - galagaalpha#0

> Upvote - freelancerjw#0

> Upvote - dalgam#0

> Upvote - Wismerhill#2123

> Upvote - doctor_medic#0

> Upvote - fallenone121#0

> Upvote - luscatore#0

> Upvote - .inchy#0

> Upvote - tackti#0

> Upvote - shaneisgay3428#0

> Upvote - thydude#0

> Downvote - dgjbfd#0

> Downvote - warthog_smith#0

> Upvote - reyakthehunter#0

> Upvote - martyrcomplex#0

> Downvote - kregularjoe#0

> Upvote - freakythedeadfox#0

> Upvote - 406616129772191754

> Upvote - GrueWithNoLight#8150

> Upvote - solaire_goshaven#0

> Upvote - Larry Hoy#5790

> Upvote - dunec#0

> Upvote - 423807483811987456

> Upvote - lamar0420#0

> Upvote - aspectsr#0

> Upvote - 176019893743910913

> Upvote - ._ovo_.#0

> Upvote - 270515508183433216

> Upvote - divinitusrenes#0

> Upvote - smashman#0

> Upvote - 446097079560044544

> Upvote - 131976201597222912

> Upvote - jonobi#0

> Upvote - jolanarr#0

> Upvote - 330204767303368705

> Upvote - 627274021355913227

> Upvote - orlimalev#0

> Upvote - katlamos#0

> Upvote - 622229631894028318

> Upvote - harcagnel#0

> Upvote - kerato_san#0

> Upvote - whimsicaliber#0

> Downvote - malkavthemad#0

> Upvote - seeme11#0

> Upvote - .norl#0

> Upvote - nod_hero#0

> Upvote - tlov#0

> Upvote - rodahunts#0

> Upvote - twilight2k#0

> Upvote - aliqux#0

> Upvote - nicole_kimono#0

> Upvote - lavablade_#0

> Upvote - helle_xx#0

> Upvote - _georgek#0

> Upvote - loveclown#0

> Upvote - 349821978615742491

> Upvote - kevc2181#0

> Upvote - dobie97#0

> Upvote - airgr0und#0

> Upvote - kirby7830#0

> Upvote - dasheldartagnan#0

> Upvote - miztermeatmissile#0

> Upvote - 346897510771458048

> Upvote - 329396538876887044

> Upvote - i_c_a_r_u_s#0

> Upvote - cyberwraithe#0

> Upvote - spineoftheworld#0

> Upvote - 140929582114144256

> Upvote - aloadingscreen#0

> Upvote - bakingviking#0

> Upvote - zitterberg#0

> Upvote - random.cat#0

> Upvote - jclements#0

> Upvote - maxjohansen#0

> Upvote - 238622172908617729

> Upvote - jumperredux#0

> Upvote - ghreti#0

> Upvote - kenkanemi#0

> Upvote - _collision_#0

> Upvote - blerdspeak#0

> Upvote - asmodous#0

> Upvote - metacomet10#0

> Upvote - wessel0241#0

> Upvote - l0lki#0

> Upvote - dickboob#0

> Upvote - wcdevil#0

> Upvote - wizardsrule#0

> Upvote - cursedquail#0

> Upvote - dm_forever#0

> Upvote - kestrelkate#0

> Upvote - showdoglq#0

> Upvote - jkat718#0

> Upvote - BEQR#7003

> Upvote - _pentacore#0

> Upvote - khanbad#0

> Upvote - khaldryn#0

> Upvote - astronimos#0

> Upvote - rocky0000#0

> Upvote - golitay#0

> Upvote - golitay#0

> Indifferent - mick.orc#0

> Indifferent - chocobo108#0

> Upvote - axeofeastmarch#0

> Downvote - twilight2k#0

> Downvote - gothamdankknight#0

> Indifferent - whimsicaliber#0

> Upvote - docmikey#0

> Downvote - mchillster#0

> Downvote - divinitusrenes#0

> Indifferent - rocky0000#0

> Upvote - singady#0

> Indifferent - t.j.l.#0

> Upvote - lamar0420#0

> Downvote - 355589886486642689

> Upvote - glory_03#0

> Upvote - jaeonasi#0

> Upvote - draiegen#0

> Upvote - miztermeatmissile#0

> Upvote - joshuaas#0

> Upvote - ausren#0

Votes: +131 / -9 / ±0

**Comment by darovic** [comment]
This would be a venerable nightmare. There are just shy of 500 spells in 5E, with more being added every year. Worse, they all have varying description lengths and formats. To put that into perspective, there are about 620 magic items, including all the enhancement bonus variants. This is a lot of work for very, very little reward, especially as the spells are already there to be selected.

It just needs to be kept in perspective. If this feature is implemented, it's a longer term project than most, and would benefit greatly from community input.

**Comment by morepurplemorebetter** [comment]
I agree with @darovic, this would not be an easy feature to implement. Every spell's full description would have to be split into lines that fit the spell sheet, a dumb amount of work even when it can be partially automated.

I feel a better solution would be to have a different spell sheet format, which is not a table like the current one but, for example, has spell cards. One way or another it wouldn't just be "Spellsheets would be longer in general", but going by 12 'cards' per page compared to the current average of 67 lines per page, it would increase the page number by a factor of 6!

Don't forget that you can already view full spell descriptions by clicking the button in the relevant spell line. Of course, this isn't very helpful for those who print the sheet.

**Comment by crawltracker** [comment]
Note - morepurplemorebetter#1979

Merging "MFR-9 3x3 grid Spell sheet Option" with this request, as they are essentially the same thing.
**How would it work?** Format spell sheet more like cards from 4E with all (or as much as would fit) pertinent info paraphrased from the source.
**Why should this be added?** Many people now have issues using the spell sheets as is because of the abbreviated info box. This would also allow people to cut the sheets and put their spells into multi-up card holders sold for card collecting hobbyists.

**Comment by crawltracker** [comment]
Note - IssueCrawler#1030

Merged `MFR-118 (flapkan/mpmb-tracker#173)` into `MFR-005 (flapkan/mpmb-tracker#5)`
